src/main/scala/cromwell/binding/Call.scala

```diff
@@ -56,15 +58,27 @@ case class Call(alias: Option[String], task: Task, inputMappings: Map[String, Wd
    * of a prerequisite call), that prerequisite call will appear only once in the output.
    */
   def prerequisiteCalls(): Set[Call] = {
-    val allCalls = parent.get.asInstanceOf[Workflow].calls
-
-    val prerequisiteCalls = for {
+    inputMappings.values.flatMap {expr =>
```

Comment:
What about something like this, renaming `foo` to something appropriate?

    def prerequisiteCalls(): Set[Call] = {
      val memberAccesssAsts = for (
        expr <- inputMappings.values;
        memberAccessAst <- WdlBinding.findTopLevelMemberAccesses(expr.ast);
        memberAccessAsts <- foo(memberAccessAst)
      ) yield memberAccessAsts

     memberAccesssAsts.toSet
    }

    def foo(memberAccesssAst: WdlParser.Ast): Seq[Call] = {
      println(s"-- START partial eval $name")
      val r = binding.partialEvalMemberAccess(memberAccesssAst) match {
        case Success(c:Call) => Seq(c)
        case _ => Seq()
      }
     println(s"-- END partial eval $name")
     println(s"-- RESULT: $r")
     r
    }

Also I think you can replace this block (regardless of structure above):

    val r = binding.partialEvalMemberAccess(memberAccesssAst) match {
      case Success(c:Call) => Seq(c)
      case _ => Seq()
    }

to something like:

    binding.partialEvalMemberAccess(memberAccesssAst) map {Seq(_)} getOrElse Seq()